ts/Extensions/Annotations/Types/BasicAnnotation.ts

```diff
@@ -348,6 +348,11 @@ class BasicAnnotation extends Annotation {
         if (options.shapes) {
             delete options.labelOptions;
             const type = options.shapes[0].type;
+
+            options.shapes[0].className = pick(
```

Comment:
No need for `pick` in this case. Since `className` is `string | undefined`, we can just do `(options.shapes[0].className || '') + ' highcharts-basic-shape'`.